pkg/testcase/restartservice.go

```diff
@@ -0,0 +1,22 @@
+package testcase
+
+import (
+	"github.com/rancher/distros-test-framework/factory"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestRestartService() {
+	product, err := shared.GetProduct()
+	Expect(err).NotTo(HaveOccurred(), "failed to get product")
+	c := factory.ClusterConfig(GinkgoT())
+
+	var ip string
+	ip, err = shared.ManageService(product, "restart", "server", c.ServerIPs)
+	Expect(err).NotTo(HaveOccurred(), "failed to get systemctl command for ip %s", ip)
```

Comment:
NIT: this should be "failed to restart service for server"

This will almost never be a get systemctl cmd issue. - which would have happened only when the function call arguments are wrong. the only time this will actually fail, is when the execution of runcommand fails dynamically during run time. - and in this case it will be becuase the restart failed for that particular node.

Reply:
fixed